osgi.specs/docbook/140/service.http.whiteboard.xml

```diff
@@ -29,8 +29,8 @@
   <title>Http Whiteboard Specification</title>
 
   <info>
-    <releaseinfo><xref endterm="org.osgi.service.http.whiteboard-version"
-    linkend="org.osgi.service.http.whiteboard"/></releaseinfo>
+    <releaseinfo><xref endterm="org.osgi.service.servlet.whiteboard-version"
```

Comment:
We should change the chapter/xml:id to "service.servlet". Also the file name of this file should be "service.servlet.xml".